sentry/src/test/java/io/sentry/SentryOptionsTest.kt

```diff
@@ -114,6 +114,51 @@ class SentryOptionsTest {
         assertFailsWith<IllegalArgumentException> { SentryOptions().tracesSampleRate = -0.0000000000001 }
     }
 
+    @Test
```

Comment:
`m`: what if we reuse the test data from the PHP sdk PR and create a new Parameterized class test, so it's 100% aligned?

Reply:
Added a new test called `SentryOptionsTracingTest`